android/app/build.gradle

```diff
@@ -130,6 +132,7 @@ android {
         applicationId "com.mattermost.rnbeta"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
+        missingDimensionStrategy "RNNotifications.reactNativeVersion", "reactNative60"
```

Comment:
had to upgrade react-native-notifications so it compiles on Android (there is a ticket to upgrade to v3+)